test/conformance/tests/sriov_operator.go

```diff
@@ -1489,7 +1489,7 @@ func isInterfaceSlave(ifcPod *k8sv1.Pod, ifcName string) (bool, error) {
 		parts := strings.Split(line, " ")
 		if len(parts) > 1 && parts[1] == ifcName {
 			if strings.Index(line, "master") != -1 { // Ignore hw bridges
-				return false, nil // The interface is part of a bridge (it has a master)
+				return true, nil // The interface is part of a bridge (it has a master)
```

Comment:
so maybe you can get rid of that if entirely?
or at least correct the comment above too
we're now looking for the bridges here..

Reply:
sorry, I miss read it
now I see. returning false in that case was wrong, it is exactly when it is actually a slave and should return true :-)